src/codeswitch/data_generation/utterance_generation.py

```diff
@@ -0,0 +1,55 @@
+import json
+from pathlib import Path
+
+import openai
+import pandas as pd
+from tqdm import tqdm
+
+from codeswitch.data_generation.generate_prompt_template import CONTEXTS, get_template
+
+# Load OpenAI API key from environment variable
+# openai.api_key = os.getenv("open_ai_key")
+# Load the prompts DataFrame
+df = pd.read_csv("generated_prompts.csv")
+
+
+# Function to generate code-switched utterances for a given prompt
+def generate_code_switched_utterance(prompt):
+    prompt += """
+    Give me a list of generated sentences in the form of {'sentences': ['sentence 1', 'sentence 2']}
+    """
+    response = openai.ChatCompletion.create(
+        model="gpt-3.5-turbo",
+        response_format={"type": "json_object"},
+        max_tokens=200,
+        messages=[
+            {
+                "role": "system",
+                "content": "You are a helpful assistant designed to output JSON.",
```

Comment:
You can mention the overall goal of the task here. For example "You are a helpful assistant designed to output code-switched sentences in the format of a JSON." This is just a quick example. Try to come up with something better.